### DISCUSSION TITLE
Error when running coolprop script: "expected bytes, str found"

### DISCUSSION THREAD

**Discussion by ibuzzo** [CoolProp/CoolProp#2327] [Q&A]
I have just installed Spyder and enhaced it with python 3.12.0. I have no problem when running a  pandas/numpy scrpit, but with coolprop i have this issue. Obviosuly , i have previously installed coolprop from spyder terminal.

![Coolprop error](https://github.com/CoolProp/CoolProp/assets/152515112/5a854360-3709-4218-b5da-6636f2e8a352)

**Comment by fwitte** [comment]
Hi,

I'd recommend you post the respective code section here that causes that error. I can guess based on what I have seen before: you may have forgotten to add the fluid name in the PropsSI call. (The error message for that kind of issue is not really pointing people in the right direction...)

Best

**Reply by ibuzzo** [reply]
Hi, thanks for answering. Here is the code:

![image](https://github.com/CoolProp/CoolProp/assets/152515112/3f74346d-40f9-4be3-b1fa-aff78dc020ec)

**Comment by ibell** [comment]
What operating system are you using?
